lib/mix/tasks/elasticsearch_setup.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Mix.Tasks.ElasticsearchSetup do
+  use Mix.Task
+  require Tesla
+
+  @elastic_cluster_url Application.compile_env(:elasticsearch_test, :cluster_url)
```

Comment:
think these files might be git leftovers (I moved them to a folder under `mix/tasks/elasticsearch`